### PR TITLE
Implementing baseline U-Net based on PyTorch. 

### PR DESCRIPTION
**Pull request by Aavache**
# 🚀 Purpose of this PR

Currently, there is only one dummy algorithm to detect cells in the repository. It would be useful for participants to have another reference baseline on how to run algorithms with a popular deep learning framework leveraging the power of GPUs.

The proposed model (U-Net) model was pre-trained with the OCELOT training set. You will find the required reproducibility documentation under `user/unet_example/README.md`. 

Please, be aware that this method is a cell-only baseline, therefore it does not consider the larger tissue patch context.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +93,9 @@ def process_patch_pair(cell_patch, tissue_patch, pair_id, meta_dataset):
     return list(zip(xs, ys, class_id, probs))
 
 ```
+
+For more inference examples, please visit `/ocelot23algo/user/unet_example/`.
```
Comment (rjw0205) [inline README.md]: Can we make this sentence more detailed? 

--> To guide participants with the actual example, we provide a simple baseline algorithm that uses the U-Net architecture. For more details, please visit [ocelot23algo/user/unet_example/](https://github.com/lunit-io/ocelot23algo/tree/feature/unet-example/user/unet_example).

**Review by rjw0205 — CHANGES_REQUESTED** [review]
Thanks for your hard effort, I only left one comment!

File: README.md (hunk 1 above)
Comment (Aavache) [reply README.md]: Thank you for the comment @rjw0205! I applied the change.

**Review by rjw0205 — APPROVED** [review]
Thanks for your effort, LGTM!

**Pull request merged**